app/models/manageiq/providers/azure/inventory/collector.rb

```diff
@@ -262,14 +262,14 @@ def instances
   # that it doesn't affect the rest of inventory collection.
   #
   def images
-    collect_inventory(:private_images) { gather_data_for_this_region(@sas, 'list_all_private_images') }
+    collect_inventory(:private_images) { @sas.list_all_private_images(:location => @ems.provider_region) }
```

Comment:
Does this need a filter_my_region call?

Reply:
No because this call accepts a `:location` option directly we don't need it, see the old call https://github.com/ManageIQ/manageiq-providers-azure/pull/522/files#diff-8af42162f4f68708edd4edd030961533485f12f82576d34691847b9cef1f91dcL73-L74